pymc/distributions/multivariate.py

```diff
@@ -565,13 +565,14 @@ def dist(cls, n, p, *args, **kwargs):
     def get_moment(rv, size, n, p):
         if p.ndim > 1:
             n = at.shape_padright(n)
-        if (p.ndim == 1) & (n.ndim > 0):
+        if (p.ndim == 1) and (n.ndim > 0):
             n = at.shape_padright(n)
             p = at.shape_padleft(p)
```

Comment:
I think we shouldn't need these shape_padright/left anymore

Reply:
I tried removing
```py
 if p.ndim > 1:
            n = at.shape_padright(n)
        if (p.ndim == 1) and (n.ndim > 0):
            n = at.shape_padright(n)
            p = at.shape_padleft(p)
```

but existing tests fail in `test_distributions_moments.py`

```
FAILED test_distributions_moments.py::test_multinomial_moment[p4-n4-None-expected4]
FAILED test_distributions_moments.py::test_multinomial_moment[p5-n5-None-expected5]
FAILED test_distributions_moments.py::test_multinomial_moment[p6-n6-2-expected6]
```